crates/wdk-build/src/cargo_make.rs

```diff
@@ -1052,6 +1062,38 @@ where
     env::set_var(env_var_name, env_var_value);
 }
 
+/// `cargo-make` condition script for `infverif` task in
+/// [`rust-driver-sample-makefile.toml`](../rust-driver-sample-makefile.toml)
+///
+/// # Errors
+///
+/// This function returns an error whenever it determines that the
+/// `infverif` `cargo-make` task should be skipped (i.e. when the WDK Version is
+/// bugged and does not contain /samples flag
+///
+/// # Panics
+/// Panics if the `WDK_BUILD_DETECTED_VERSION` environment variable is not set.
```

Comment:
This actually only panics if `CARGO_MAKE_CURRENT_TASK_NAME` is not set. See `condition_script` function's docs